### ISSUE TITLE
MockedFileSystem doesnt handle Directory attribute (DirectoryExists works though)

### ISSUE THREAD

**Issue by Rayvid** [tathamoddie/System.IO.Abstractions#93]
Will you accept mergerequest with fix?

**Comment by ericnewton76** [comment]
Uh, what merge request are you talking about?

**Comment by Rayvid** [comment]
Hypothetical - if i would make one :)?

G.

**Comment by Holf** [comment]
Just noticed this myself. With file system:

    var fileSystem = new MockFileSystem(new Dictionary<string, MockFileData>
    {
        {@"\\Folder\log1.log", new MockFileData( "Log1.")},
        {@"\\Folder\log2.log", new MockFileData( "Log2.")}
    });

I would expect:

    FileAttributes attr = _fileSystem.File.GetAttributes("\\Folder\");
    if (attr.HasFlag(FileAttributes.Directory))
    ....

...to be true.

(Amazing library, by the way.)

**Comment by manne** [comment]
@Holf your code throws on my computer with the current version an `ArgumentException` with the message `"The UNC path should be of the form \\server\share.Parameter name: path"`. Do you have the current version?

However
```
	var fs = new MockFileSystem(new Dictionary<string, MockFileData>
	{
		{@"\\Folder\aa\log1.log", new MockFileData( "Log1.")},
		{@"\\Folder\aa\log2.log", new MockFileData( "Log2.")}
	});
	
	fs.File.GetAttributes(@"\\Folder\aa\").Dump(); // => Normal
```
Should return `FileAttributes.Directory`, but does return `FileAttributes.Normal`

**Comment by manne** [comment]
see #117

**Comment by manne** [comment]
@Rayvid is your issue fixed with #118?

**Comment by Rayvid** [comment]
Y thanx a bunch!

On Wednesday, March 4, 2015, Manuel Pfemeter <notifications@github.com>
wrote:
> @Rayvid is your issue fixed with #118?
>
> —
> Reply to this email directly or view it on GitHub.<
https://ci6.googleusercontent.com/proxy/id34JdXd69HHxh0LdXKqvJxoC_KBWD0-uz0ySQzkdsXQ-6Z5XiF_iMvO7u-03sXJg86IzUKSM4s3lmT-MC_NNJq9qrUDzr5CPEATeFf14NrHXqoBLGT5AmFVHgxSlIZq1s7U6NxtvF5Jb4uLY1lgeX0_U6LirA=s0-d-e1-ft#https://github.com/notifications/beacon/AA1py63oagoeOsxZlZUUIC2F7NcfQat9ks5nx2dkgaJpZM4DOUFW.gif
>

**Comment by manne** [comment]
@rayvid can you close the issue
